doc/examples/transform/plot_radon_transform.py

```diff
@@ -98,12 +98,30 @@
 # back projection is among the fastest methods of performing the inverse
 # Radon transform. The only tunable parameter for the FBP is the filter,
 # which is applied to the Fourier transformed projections. It may be used to
-# suppress high frequency noise in the reconstruction. ``skimage`` provides a
-# few different options for the filter.
+# suppress high frequency noise in the reconstruction. ``skimage`` provides
+# the filters 'ramp', 'shepp-logan', 'cosine', 'hamming', and 'hann':
+
+import matplotlib.pyplot as plt
+from skimage.transform.radon_transform import _get_fourier_filter
+
+filters = ['ramp', 'shepp-logan', 'cosine', 'hamming', 'hann']
+
+for ix, f in enumerate(filters):
+    response = _get_fourier_filter(2000, f)
+    plt.plot(response, label= f)
+
+plt.xlim([0, 1000])
+plt.xlabel('frequency')
+plt.legend()
+plt.show()
+
+######################################################################
+# Applying the inverse radon transformation with the 'ramp' filter, we get:
 
 from skimage.transform import iradon
 
-reconstruction_fbp = iradon(sinogram, theta=theta, circle=True)
+reconstruction_fbp = iradon(sinogram, theta=theta, filter_name='ramp',
+                            circle=True)
```

Comment:
```suggestion
reconstruction_fbp = iradon(sinogram, theta=theta, filter_name='ramp')
```
I know it's not related to this PR specifically, but I don't see the point of specifying `circle=True`, since it's the default value for this parameter *and* the narrative parts of the example do not refer to this parameter. Poking @rfezzani just in case.